go-controller/pkg/libovsdbops/loadbalancer.go

```diff
@@ -220,3 +221,32 @@ func ListLoadBalancers(nbClient libovsdbclient.Client) ([]nbdb.LoadBalancer, err
 	}
 	return *lbs, nil
 }
+
+// FindLoadBalancersByEcternalIDs find all load balancers whose ExternalIDs contain at least the
```

Comment:
@squeed `Ecternal` ?

Reply:
Hah, oops. New fancy mechanical keyboard. I'm still getting used to it...